bridge/slack/handlers.go

```diff
@@ -90,248 +98,170 @@ func (b *Bslack) handleMatterHook(messages chan *config.Message) {
 	}
 }
 
-var commentRE = regexp.MustCompile(`.*?commented: (.*)`)
-
-// handleDownloadFile handles file download
-func (b *Bslack) handleDownloadFile(rmsg *config.Message, file *slack.File) error {
-	// if we have a file attached, download it (in memory) and put a pointer to it in msg.Extra
-	// limit to 1MB for now
-	comment := ""
-	results := commentRE.FindAllStringSubmatch(rmsg.Text, -1)
-	if len(results) > 0 {
-		comment = results[0][1]
-	}
-	err := helper.HandleDownloadSize(b.Log, rmsg, file.Name, int64(file.Size), b.General)
-	if err != nil {
-		return err
+// skipMessageEvent skips event that need to be skipped :-)
+func (b *Bslack) skipMessageEvent(ev *slack.MessageEvent) bool {
+	switch ev.SubType {
+	case sChannelLeave, sChannelJoin:
+		return b.GetBool(noSendJoinConfig)
+	case sPinnedItem, sUnpinnedItem:
+		return true
 	}
-	// actually download the file
-	data, err := helper.DownloadFileAuth(file.URLPrivateDownload, "Bearer "+b.GetString(tokenConfig))
-	if err != nil {
-		return fmt.Errorf("download %s failed %#v", file.URLPrivateDownload, err)
+
```

Comment:
This additional `ev.Username == sSlackBotUser` check below should be a fix for #527 (cc @patcon).

Reply:
Thanks! Should all ephemeral messages from bots be ignored? Or even all ephemeral messages? to the bot/slack user?